ant-networking/src/lib.rs

```diff
@@ -406,6 +408,10 @@ impl Network {
             .send_and_get_responses(&close_nodes, &request, true)
             .await;
 
+        // consider data to be already paid for if 1/3 of the close nodes already have it
+        let mut peer_already_have_it = 0;
+        let enough_peers_already_have_it = close_nodes.len() / 3;
```

Comment:
to avoid malicious attacke, better at least have `2` holders.
for `close_nodes.len() = 5`, `enough_peers_already_have_it` will have a value of `1`, which is somehow vulunerable.
maybe make it `std::cmp::max(2, close_nodes.len()/3)` or `close_nodes.len() / 2` instead.

Reply:
Make sense let's go with: `close_nodes.len() / 2`